src/pages/StatisticsChart/activities/ContractResourceDistributed.tsx

```diff
@@ -82,7 +91,21 @@ const useOption = (
         },
       },
     ],
-    // TODO: add visual map when txs/24h is ready
+    visualMap: [
+      {
+        min: h24TxCountSortedList[-1] === undefined ? 0 : h24TxCountSortedList[-1],
```

Comment:
`h24TxCountSortedList[-1]` should be `h24TxCountSortedList[h24TxCountSortedList.length - 1]` because `h24TxCountSortedList[-1` is always `undefined`

Reply:
fix